etl/ems_person_match/utils/match_rules.py

```diff
@@ -0,0 +1,27 @@
+"""
+Hierarchical matching rules for linking EMS PCR records to CRIS people records.
+
+Rules are applied sequentially from most to least restrictive. This ensures
+high-confidence matches are made first.
+
+Each rule specifies which attributes must match  between the
+PCR and person record for the pairing to be accepted. The first rule that finds
+a match wins, preventing the same person from being matched to multiple PCRs.
+"""
+
+# fmt: off
+MATCH_RULES = [
```

Comment:
These match rules are very much WIP. The most permissive rules will almost definitely be removed once we look at the data with Joel and Xavier.